client/types/types.d.ts

```diff
@@ -4,7 +4,8 @@ export type LocationProps = {
 }
 
 export type BikeProp = {
```

Comment:
This might need a different name in the future since it goes beyond just a bike.

Reply:
Totally. Maybe we could chat further about our type declarations on Wednesday.